docs/actions/filter-grid.md

```diff
@@ -0,0 +1,35 @@
+---
+id: filter-action-grid
+title: Filter Action Grid
+sidebar_label: Filter Action Grid
+---
+
+> Audience: [`Citizen Developers`](/docs/audience#citizen-developers)<br/>
+> Skill Prerequisites: `Using Action Grid`
+> 
+This action refreshes an Action Grid module without page reload (if it’s on same page) with the option to change sorting, pagination and filtering.
+
+## `Typical Use Cases`
+
+- Use an Action Form to filter and/or sort an Action Grid
+
+
+## `Input Parameter Reference`
+
+| Parameter | Description | Supports Tokens | Default | Required |
+| -- | -- | -- | -- | -- |
+| Select Module | Select the Grid Module you wish to filter| Yes | None Selected| No |
+| Page Size | Select the number of elements(rows) that will be shown in grid| No | None | No |
+| Sort By | Select the field to sort by. Ascending is the default but you can select Descending using the checkbox| Yes | Ascending | No |
+| Search Terms| Grid will automatically search for the characters passed by this field| Yes | Unchecked | No |
+

```

Comment:
Even though they aren't included on the current DNN Sharp documentation, I think we should go ahead and add the **Current Page**, **Filters**, **Replace Filters**, and **Additional Data** parameters here.

Reply:
Hi Linda, 

This review process is frustrating, I know! I can't tell whether you added these additional params or not. If not, I can add them, but I don't want to redo something you already did or may be in process. 

Let me know, 
-Patrick

Reply:
@pande3 I believe I have added the additional parameters so you can see them now.  Let me know if it isn't right.